rotkehlchen/data_import/utils.py

```diff
@@ -75,9 +118,44 @@ def flush_all(self, write_cursor: DBCursor) -> None:
         self._asset_movements = []
         self._history_events = []
 
+    def append_msg(self, row_index: int, msg: str, warning: bool) -> None:
+        """Append message to queue to be sent to frontend"""
```

Comment:
Move the comment description of the MAX_ERROR_PERCENT and explain the logic of what you are doing here in the docstring